asrtoolkit/file_utils/name_cleaners.py

```diff
@@ -44,7 +44,7 @@ def replace_char(input_char, file_name):
                 "check to make sure your audio files and transcript files match"
             )
         return os.path.join(
-            file_name.split(os.sep)[:-1] +
+            *file_name.split(os.sep)[:-1] +
```

Comment:
If you're trying to get the file directory name here you could use `os.path.dirname()`? It's more self documenting.